<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 3 An application to Statistical Genetics | Hidden Markov Models &amp; their Applications to Statistical Genetics</title>
  <meta name="description" content="Chapter 3 An application to Statistical Genetics | Hidden Markov Models &amp; their Applications to Statistical Genetics" />
  <meta name="generator" content="bookdown 0.21 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 3 An application to Statistical Genetics | Hidden Markov Models &amp; their Applications to Statistical Genetics" />
  <meta property="og:type" content="book" />
  
  
  
  <meta name="github-repo" content="freddy-barragan/455_Markov" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 3 An application to Statistical Genetics | Hidden Markov Models &amp; their Applications to Statistical Genetics" />
  
  
  

<meta name="author" content="Freddy Barragan, Spring 2021, Macalester College" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="frog-families-hmms.html"/>

<script src="libs/header-attrs-2.7/header-attrs.js"></script>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />












<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Hidden Markov Models</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Welcome</a></li>
<li class="chapter" data-level="1" data-path="mathematical-intuition.html"><a href="mathematical-intuition.html"><i class="fa fa-check"></i><b>1</b> Mathematical Intuition</a>
<ul>
<li class="chapter" data-level="1.1" data-path="mathematical-intuition.html"><a href="mathematical-intuition.html#conditional-probability-bayes-rule"><i class="fa fa-check"></i><b>1.1</b> Conditional Probability &amp; Bayes Rule</a>
<ul>
<li class="chapter" data-level="" data-path="mathematical-intuition.html"><a href="mathematical-intuition.html#basic-concepts"><i class="fa fa-check"></i>Basic Concepts</a></li>
<li class="chapter" data-level="" data-path="mathematical-intuition.html"><a href="mathematical-intuition.html#conditional-probabilities"><i class="fa fa-check"></i>Conditional Probabilities</a></li>
<li class="chapter" data-level="" data-path="mathematical-intuition.html"><a href="mathematical-intuition.html#bayes-rule-lotp"><i class="fa fa-check"></i>Bayes’ Rule &amp; LOTP</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="mathematical-intuition.html"><a href="mathematical-intuition.html#markov-chains"><i class="fa fa-check"></i><b>1.2</b> Markov Chains</a>
<ul>
<li class="chapter" data-level="" data-path="mathematical-intuition.html"><a href="mathematical-intuition.html#intuition"><i class="fa fa-check"></i>Intuition</a></li>
<li class="chapter" data-level="" data-path="mathematical-intuition.html"><a href="mathematical-intuition.html#mathematical-definitions"><i class="fa fa-check"></i>Mathematical Definitions</a></li>
<li class="chapter" data-level="" data-path="mathematical-intuition.html"><a href="mathematical-intuition.html#visualizations"><i class="fa fa-check"></i>Visualizations</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="mathematical-intuition.html"><a href="mathematical-intuition.html#conclusion"><i class="fa fa-check"></i><b>1.3</b> Conclusion</a></li>
<li class="chapter" data-level="" data-path="mathematical-intuition.html"><a href="mathematical-intuition.html#video-resources"><i class="fa fa-check"></i>Video Resources</a></li>
<li class="chapter" data-level="" data-path="mathematical-intuition.html"><a href="mathematical-intuition.html#references"><i class="fa fa-check"></i>References</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="frog-families-hmms.html"><a href="frog-families-hmms.html"><i class="fa fa-check"></i><b>2</b> Frog Families &amp; HMMs</a>
<ul>
<li class="chapter" data-level="2.1" data-path="frog-families-hmms.html"><a href="frog-families-hmms.html#hidden-markov-models"><i class="fa fa-check"></i><b>2.1</b> Hidden Markov Models</a>
<ul>
<li class="chapter" data-level="" data-path="frog-families-hmms.html"><a href="frog-families-hmms.html#properties"><i class="fa fa-check"></i>Properties</a></li>
<li class="chapter" data-level="" data-path="frog-families-hmms.html"><a href="frog-families-hmms.html#example-frogs"><i class="fa fa-check"></i>Example: Frogs</a></li>
<li class="chapter" data-level="" data-path="frog-families-hmms.html"><a href="frog-families-hmms.html#visualizations-1"><i class="fa fa-check"></i>Visualizations</a></li>
</ul></li>
<li class="chapter" data-level="2.2" data-path="frog-families-hmms.html"><a href="frog-families-hmms.html#likelihood-the-forward-backward-algorithm"><i class="fa fa-check"></i><b>2.2</b> Likelihood &amp; The Forward-Backward Algorithm</a>
<ul>
<li class="chapter" data-level="" data-path="frog-families-hmms.html"><a href="frog-families-hmms.html#likelihood"><i class="fa fa-check"></i>Likelihood</a></li>
<li class="chapter" data-level="" data-path="frog-families-hmms.html"><a href="frog-families-hmms.html#the-forward-backward-algorithm"><i class="fa fa-check"></i>The Forward-Backward Algorithm</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="frog-families-hmms.html"><a href="frog-families-hmms.html#conclusion-1"><i class="fa fa-check"></i><b>2.3</b> Conclusion</a></li>
<li class="chapter" data-level="" data-path="frog-families-hmms.html"><a href="frog-families-hmms.html#video-resources-1"><i class="fa fa-check"></i>Video Resources</a></li>
<li class="chapter" data-level="" data-path="frog-families-hmms.html"><a href="frog-families-hmms.html#references-1"><i class="fa fa-check"></i>References</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="an-application-to-statistical-genetics.html"><a href="an-application-to-statistical-genetics.html"><i class="fa fa-check"></i><b>3</b> An application to Statistical Genetics</a>
<ul>
<li class="chapter" data-level="3.1" data-path="an-application-to-statistical-genetics.html"><a href="an-application-to-statistical-genetics.html#local-ancestry-inference"><i class="fa fa-check"></i><b>3.1</b> Local Ancestry Inference</a>
<ul>
<li class="chapter" data-level="" data-path="an-application-to-statistical-genetics.html"><a href="an-application-to-statistical-genetics.html#local-ancestry"><i class="fa fa-check"></i>Local Ancestry</a></li>
<li class="chapter" data-level="" data-path="an-application-to-statistical-genetics.html"><a href="an-application-to-statistical-genetics.html#hapmix"><i class="fa fa-check"></i>HAPMIX</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="an-application-to-statistical-genetics.html"><a href="an-application-to-statistical-genetics.html#conclusion-more"><i class="fa fa-check"></i>Conclusion &amp; More</a></li>
<li class="chapter" data-level="" data-path="an-application-to-statistical-genetics.html"><a href="an-application-to-statistical-genetics.html#references-2"><i class="fa fa-check"></i>References</a></li>
</ul></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Hidden Markov Models &amp; their Applications to Statistical Genetics</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="an-application-to-statistical-genetics" class="section level1" number="3">
<h1><span class="header-section-number">Chapter 3</span> An application to Statistical Genetics</h1>
<p>The goals of epidemiology explicitly revolve around studying the incidence, distribution, and control of disease. As one can imagine, the boundaries of the field are highly nebulous. So far, we’ve looked at Hidden Markov Models as mathematical objects, devoid of any real context— aside from a toy frog example. This section intends to tie together the mathematical principals of HMMs with their connections to big areas in biostatistical &amp; epidemiological research. We find that, yes, HMMs <em>are</em> cool because they can help save lives, bridge the gaps across survival disparities, and study our bodies.</p>
<p><br />
</p>
<hr />
<div id="local-ancestry-inference" class="section level2" number="3.1">
<h2><span class="header-section-number">3.1</span> Local Ancestry Inference</h2>
<p>Local Ancestry Inference (LAI) is an approach in statistical genetics &amp; bioinformatics that aims to study genetic diversity, disease prevalence, or evolutionary development by linking each segment of a genome to some ancestral population<a href="#fn1" class="footnote-ref" id="fnref1"><sup>1</sup></a>. LAI can been used to study crop plant analysis<a href="#fn2" class="footnote-ref" id="fnref2"><sup>2</sup></a>, Tilapia breeding<a href="#fn3" class="footnote-ref" id="fnref3"><sup>3</sup></a>, and the genetic adaptations of cows in East Africa<a href="#fn4" class="footnote-ref" id="fnref4"><sup>4</sup></a>. However, using LAI in public health studies is a critically important step, when studying ethnically-diverse cohorts.</p>
<p>Diasporic movement, mass-migration, colonial empire, and slavery have profoundly changed the genetic landscape of the human population. Prior to these <em>admixture events</em>, human communities were largely isolated from each other and overtime, these isolated communities genetically diverged into human sub-populations. The differences between sub-populations are largely minuscule. And while the differences are mostly nonexistent, generations of independent breeding did allow for the development of genes &amp; mutations which are more common in some sub-populations rather than others<a href="#fn5" class="footnote-ref" id="fnref5"><sup>5</sup></a>.</p>
<blockquote>
<p><strong>NOTE:</strong> Ancestry &amp; Race are two different concepts that do not measure the same thing in the same way.</p>
</blockquote>
<p>Admixed populations are products of the intense migratory &amp; diasporic events which marked the 14th-19th centuries, which resulted in populations whose genetic ancestries were from 2 or more human sub-populations. We call these ancestrally-mixed populations <strong>admixed</strong> populations. Notable examples of admixed populations include African Americans, Latines, Ashkenazi Jews, and SWANA peoples. However, admixture is everpresent across most global populations. For example, this plot made by Hodgson et al. shows the admixture proportion across 81 different African populations with colorings for 10, 11, and 12 respective common ancestries.<a href="#fn6" class="footnote-ref" id="fnref6"><sup>6</sup></a></p>
<p><br />
</p>
<center>
<div class="figure">
<img src="images/map.png" alt="" />
<p class="caption">Figure 1. Admixture proportions of Horn of Africa populations (Hodgson et al.)</p>
</div>
</center>
<p><br />
</p>
<p>Unfortunately, the violence of diaspora, colonialism, and enslavement also brought the dangers of inheriting potentially dangerous mutations by family, ancestry, or ethnicity. For example, the inheritance of Tay-Sachs, Ellis-van Creveld Syndrome, diabetes, prostate cancer all occur with strong associations to ancestry<a href="#fn7" class="footnote-ref" id="fnref7"><sup>7</sup></a>.</p>
<p>So, when studying disease with evidence of some ancestral association, LAI allows public researchers to first map the ancestry of an individual’s genome, before going on to find associations between disease &amp; ancestral lineage.</p>
<div id="local-ancestry" class="section level3 unnumbered">
<h3>Local Ancestry</h3>
<p>Local Ancestry is formally defined as “the genetic ancestry of an individual at a particular chromosomal location, where an individual can have 0, 1 or 2 copies of an allele derived from each ancestral population.<a href="#fn8" class="footnote-ref" id="fnref8"><sup>8</sup></a>”</p>
<p>Due to the structure of gene inheritance, we can model local ancestry as a Hidden Markov Model, with the observed! This idea has been formalized throughout literature on statistical genetics, but we will look at one particular approach of using HMMs in the software HAPMIX to infer local ancestry in admixed populations.</p>
</div>
<div id="hapmix" class="section level3 unnumbered">
<h3>HAPMIX</h3>
<blockquote>
<p><strong>NOTE:</strong> Most content in this section originates from HAPMIX’s official release paper by Price et al<a href="#fn9" class="footnote-ref" id="fnref9"><sup>9</sup></a>.</p>
</blockquote>
<p><strong>Definitions</strong></p>
<ul>
<li>A haplotype is a gene that is solely inherited from one parent and thus one parental population. Haplotypes are generally considered good markers of ancestral lineage.</li>
</ul>
<p><strong>Assumptions:</strong></p>
<ul>
<li>We assume that the admixed population comes from only two ancestral populations</li>
<li>We have access to reference data derived from unadmixed reference populations</li>
<li>The unadmixed reference populations are closely related to the true ancestral populations.</li>
</ul>
<p><strong>Process</strong></p>
<ol style="list-style-type: decimal">
<li><p>At every locus, HAPMIX estimates the likelihood that a “parental” haplotype from an admixed individual comes from a referenced ancestral population.</p></li>
<li><p>Loci that are next to each other are more likely to come from the same ancestral lineage<a href="#fn10" class="footnote-ref" id="fnref10"><sup>10</sup></a>, but that doesn’t hold true after enough positions. So, we say that the number of genes inherited from an ancestral population at the <span class="math inline">\(j^th\)</span> locus, can be strictly predicted by the ancestry of the <span class="math inline">\({j-1}^{th}\)</span> locus.</p></li>
<li><p>We can then use a HMM where the hidden states are the number of ancestral genes inherited <span class="math inline">\((0,1,2)\)</span> at locus <span class="math inline">\(j\)</span> and the observations are the likelihoods of observing some mutation at locus <span class="math inline">\(j\)</span>, given the ancestral reference panel.</p></li>
<li><p>Using a smoother analysis, similar to the Forward-Backward Algorithm, we can combine these likelihoods into to provide a probabilistic estimate of ancestry at each locus.</p></li>
</ol>
<p><strong>Visual</strong></p>
<p>Below is a visual made by Price et al. describing the HMM employed in HAPMIX.</p>
<center>
<div class="figure">
<img src="images/hapmix.png" alt="" />
<p class="caption">Fig 2. Markov Schematic (Price et al.)</p>
</div>
</center>
<ul>
<li>The thick bottom bar represents the true 2-state ancestral composition of an individual, the dotted line above it represents the ancestral reference panel, from which we compute likelihoods.</li>
<li>The line line represents a chromosomal segment of an individual, while the black circles represent typed mutations sites that can be distinguished between the ancestries.</li>
<li>When the yellow line crosses a red or blue line, it is visualizing the selection of an ancestral group, given the individual’s observed chromosome. Notice that when the line crosses through a dot on one particular side, it will choose the side with equal or greater dots that are vertically aligned.</li>
<li>The yellow line can misclassify the ancestries, since this is a probabilistic model. Notice the portion of blue dots above the red bar.</li>
</ul>
</div>
</div>
<div id="conclusion-more" class="section level2 unnumbered">
<h2>Conclusion &amp; More</h2>
<p>LAI is only one application of HMMs. Variations of HMMs can be used to study the differential use of condoms for HIV prevention<a href="#fn11" class="footnote-ref" id="fnref11"><sup>11</sup></a>, the race-differences in the progression of HIV for vulnerable, infected, and AIDS positive people between African American &amp; White People<a href="#fn12" class="footnote-ref" id="fnref12"><sup>12</sup></a>, or even when automatically monitoring the development of Influenza outbreaks<a href="#fn13" class="footnote-ref" id="fnref13"><sup>13</sup></a>.</p>
<p>This is only a primer (with a de-emphasis on application), but I absolutely suggest reading through the footnote references to so how cool HMMs become when using them to find ways of helping people, treating illness, and more!</p>
<p><br />
</p>
</div>
<div id="references-2" class="section level2 unnumbered">
<h2>References</h2>

</div>
</div>


















<div class="footnotes">
<hr />
<ol start="1">
<li id="fn1"><p><a href="http://cs229.stanford.edu/proj2015/290_report.pdf" class="uri">http://cs229.stanford.edu/proj2015/290_report.pdf</a><a href="an-application-to-statistical-genetics.html#fnref1" class="footnote-back">↩︎</a></p></li>
<li id="fn2"><p><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7003078/" class="uri">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7003078/</a><a href="an-application-to-statistical-genetics.html#fnref2" class="footnote-back">↩︎</a></p></li>
<li id="fn3"><p><a href="https://www.nature.com/articles/s41598-020-75744-9" class="uri">https://www.nature.com/articles/s41598-020-75744-9</a><a href="an-application-to-statistical-genetics.html#fnref3" class="footnote-back">↩︎</a></p></li>
<li id="fn4"><p><a href="https://www.frontiersin.org/articles/10.3389/fgene.2014.00443/full" class="uri">https://www.frontiersin.org/articles/10.3389/fgene.2014.00443/full</a><a href="an-application-to-statistical-genetics.html#fnref4" class="footnote-back">↩︎</a></p></li>
<li id="fn5"><p><a href="https://pubmed.ncbi.nlm.nih.gov/20594047/" class="uri">https://pubmed.ncbi.nlm.nih.gov/20594047/</a><a href="an-application-to-statistical-genetics.html#fnref5" class="footnote-back">↩︎</a></p></li>
<li id="fn6"><p><a href="http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1004393" class="uri">http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1004393</a><a href="an-application-to-statistical-genetics.html#fnref6" class="footnote-back">↩︎</a></p></li>
<li id="fn7"><p><a href="https://pubmed.ncbi.nlm.nih.gov/20594047/" class="uri">https://pubmed.ncbi.nlm.nih.gov/20594047/</a><a href="an-application-to-statistical-genetics.html#fnref7" class="footnote-back">↩︎</a></p></li>
<li id="fn8"><p><a href="https://onlinelibrary.wiley.com/doi/abs/10.1002/gepi.21819" class="uri">https://onlinelibrary.wiley.com/doi/abs/10.1002/gepi.21819</a><a href="an-application-to-statistical-genetics.html#fnref8" class="footnote-back">↩︎</a></p></li>
<li id="fn9"><p><a href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1000519" class="uri">https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1000519</a><a href="an-application-to-statistical-genetics.html#fnref9" class="footnote-back">↩︎</a></p></li>
<li id="fn10"><p>due to a concept called Linkage Disequilibrium<a href="an-application-to-statistical-genetics.html#fnref10" class="footnote-back">↩︎</a></p></li>
<li id="fn11"><p><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3649016/" class="uri">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3649016/</a><a href="an-application-to-statistical-genetics.html#fnref11" class="footnote-back">↩︎</a></p></li>
<li id="fn12"><p><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4023279/" class="uri">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4023279/</a><a href="an-application-to-statistical-genetics.html#fnref12" class="footnote-back">↩︎</a></p></li>
<li id="fn13"><p><a href="https://link.springer.com/chapter/10.1007/978-3-540-45231-7_48" class="uri">https://link.springer.com/chapter/10.1007/978-3-540-45231-7_48</a><a href="an-application-to-statistical-genetics.html#fnref13" class="footnote-back">↩︎</a></p></li>
</ol>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="frog-families-hmms.html" class="navigation navigation-prev navigation-unique" aria-label="Previous page"><i class="fa fa-angle-left"></i></a>

    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": false,
"twitter": false,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["455_Markov.pdf", "455_Markov.html"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
